src/main/java/edu/cornell/kfs/coa/businessobject/GlobalObjectWithIndirectCostRecoveryAccounts.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.coa.businessobject;
+
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+import org.kuali.rice.krad.bo.GlobalBusinessObjectDetailBase;
+
+public interface GlobalObjectWithIndirectCostRecoveryAccounts {
```

Comment:
Related to a question Chad had previously asked about this (when it was an abstract class), would it make sense for this to extend GlobalBusinessObject? SubAccountGlobal would only need to implement one interface if that was done.